webui/react/src/e2e/tests/userManagement.spec.ts

```diff
@@ -19,48 +19,95 @@ test.describe('User Management', () => {
     await expect(page).toHaveURL(userManagementPage.url);
   });
 
+  test('Users table count matches admin page users tab', async ({ page }) => {
+    const userManagementPage = new UserManagement(page);
+    await userManagementPage.goto();
+    const pagination = userManagementPage.table.table.pagination;
+    let expetedRowCount: number;
+    await test.step('Get number of users from the tab at the top', async () => {
+      const match = (await userManagementPage.userTab.pwLocator.innerText()).match(/\d+/);
+      if (match === null) {
+        throw new Error('Number not present in tab.');
+      }
+      expetedRowCount = +match[0];
+    });
+    for await (const [index, paginationOption] of [
+      pagination.perPage.perPage10,
+      pagination.perPage.perPage20,
+      pagination.perPage.perPage50,
+      pagination.perPage.perPage100,
+    ].entries()) {
+      await test.step(`Compare table rows with pagination:${index}`, async () => {
+        const match = (await userManagementPage.userTab.pwLocator.innerText()).match(/\d+/);
+        if (match === null) {
+          throw new Error('Number not present in tab.');
+        }
+        expetedRowCount = +match[0];
+        await pagination.perPage.pwLocator.click();
+        await paginationOption.pwLocator.click();
+        await expect(userManagementPage.skeletonTable.pwLocator).not.toBeVisible();
+        const matches = (await pagination.perPage.pwLocator.innerText()).match(/(\d+) \/ page/);
+        if (matches === null) {
+          throw new Error("Couldn't find pagination selection.");
+        }
+        const paginationSelection = +matches[1];
+        await expect(userManagementPage.table.table.rows.pwLocator).toHaveCount(
+          Math.min(paginationSelection, expetedRowCount),
+        );
+      });
+    }
+  });
+
   test.describe('With a new User', () => {
     let page: Page;
     let authFixture: AuthFixture;
+    let userFixture: UserFixture;
     let userManagementPage: UserManagement;
-    let userid: string;
-    let username: string;
+    let testUser: User;
 
     test.beforeAll(async ({ browser }) => {
-      username = 'test-user-' + uuidv4();
-
       await test.step('Login', async () => {
-        page = await browser.newPage();
+        page = await browser.newPage({ recordVideo: { dir: './src/e2e/test-results' } });
```

Comment:
Is this a debug thing? We should be setting up video recording in the config. I'd prefer not to have it setup on a test-by-test basis

Reply:
this is required to get video on the "second page". If I wasnt doing this `new Page` then I wouldn't need to save this video. only problem is that it doesn't show in the same folder as the test videos

Reply:
I ended up changing the tests to use the "other page" and so now I don't need to save a video from this one anymore. changes were made [here](https://github.com/determined-ai/determined/compare/jcom/INFENG-455/user-tests-continued?expand=1). i figure we can review this one separately, merge it into this PR, and merge it all into main at once